### PR TITLE
add standard SDF metrics handling and landing page tracking

### PR DESCRIPTION
**Pull request by piyalbasu**

### CODE REVIEW

File: src/constants/metricNames.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const METRIC_NAMES = {
+  viewHome: "loaded screen: home",
```
Comment (piyalbasu) [inline src/constants/metricNames.ts:2]: @shanzzam Let me know if you would like to change this label! This is the event name that will show up in Amplitude

File: src/helpers/metrics.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import throttle from "lodash/throttle";
```
Comment (piyalbasu) [inline src/helpers/metrics.ts:1]: this is lifted from Freighter/Laboratory, which each have almost this exact code. I think it's best to keep this project in parity with our other tools.

@quietbits this is a prime example of something that can go into the shared "helpers" repo we've been talking about :D

File: src/pages/Landing.tsx [hunk 3]
```diff
@@ -26,6 +28,10 @@ export const Landing = () => {
   const location = useLocation();
   const history = useHistory();
 
+  useEffect(() => {
+    emitMetric(METRIC_NAMES.viewHome);
+  }, []);
+
```
Comment (piyalbasu) [inline src/pages/Landing.tsx:34]: Carl created a custom middleware in Freighter that automatically emits an event on routing. Might be nice to do something similar in a future iteration

File: src/constants/metricNames.ts (hunk 1 above)
Comment (shanzzam) [reply src/constants/metricNames.ts:2]: This is fine. I guess as there's only one page we're ever tracking this works! We can add metrics for how the rest of the content is accessed via keypair selection later.

**Review by quietbits — APPROVED** [review]

**Pull request merged**